tools/buildsys/src/builder.rs

```diff
@@ -362,7 +348,6 @@ impl DockerBuild {
                 OutputCleanup::BeforeBuild,
             ),
             target_build_args: TargetBuildArgs::Package(PackageBuildArgs {
-                image_features,
```

Comment:
We can eliminate all of the `variant*` args too right?

Reply:
Woah, that's crazy talk. Let me try.